src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/GeneExternalReferences.tsx

```diff
@@ -91,6 +91,9 @@ const QUERY = gql`
           }
         }
         metadata {
+          biotype {
+            value
+          }
```

Comment:
Could you remind me why this was necessary? It's in EntityViewer, and neither ENSWBSITES-1191 nor ENSWBSITES-1256 explains this change.

Reply:
That's something I forgot to remove. I'll get rid of it.

Reply:
✅